README.md

```diff
@@ -452,6 +452,20 @@ In your vault go to Settings > Community plugins > Browse and search for "Execut
 or
 
 Follow [this link](https://obsidian.md/plugins?search=execute%20code#) and click "Open in Obsidian".
+	
+## Locating Path Settings ( ex. JavaScript | Node )
+	
+	To avoid or resolve errors from an incorrect path.
+	
+	1. In your terminal, type 'where node'

```

Comment:
`where` works on Windows, but not Linux. Does it work on Mac? Should we include per-OS instructions?

Reply:
`where` works on mac as well

Reply:
ahhh --- good.  I am on a mac.  I will update.  Anybody know linux command to find path?

Reply:
on Linux, it is `which` :)